src/psbt.cpp

```diff
@@ -149,6 +149,9 @@ void PSBTInput::FillSignatureData(SignatureData& sigdata) const
     for (const auto& [hash, preimage] : hash256_preimages) {
         sigdata.hash256_preimages.emplace(std::vector<unsigned char>(hash.begin(), hash.end()), preimage);
     }
+    sigdata.musig2_pubkeys.insert(m_musig2_participants.begin(), m_musig2_participants.end());
+    sigdata.musig2_pubnonces.insert(m_musig2_pubnonces.begin(), m_musig2_pubnonces.end());
+    sigdata.musig2_partial_sigs.insert(m_musig2_partial_sigs.begin(), m_musig2_partial_sigs.end());
```

Comment:
Compiling at commit 2a49082e491810fab48485e0a63e1fadfb09b1b7 currently fails, as the `musig2_`... fields haven't been added to the `SignatureData` struct yet.

Reply:
Fixed